tools/utils/fiber-helpers.js

```diff
@@ -31,26 +46,13 @@ function disallowedYield() {
 disallowedYield.disallowed = true;
 
 exports.noYieldsAllowed = function (f, context) {
```

Comment:
This function is used to wrap parts of the Meteor tool where it is potentially unsafe to have async code. Since we are switching from fibers to promises, we probably should replace it with a version that make sure no promises or async/await are used.